packages/commonwealth/client/scripts/views/components/sidebar/index.tsx

```diff
@@ -23,9 +30,18 @@ type SidebarAttrs = {
 export class Sidebar implements m.ClassComponent<SidebarAttrs> {
   view(vnode: m.VnodeDOM<SidebarAttrs, this>) {
     const { isSidebarToggleable, isSidebarToggled } = vnode.attrs;
-
-    const hideChat = !app.chain?.meta?.chatEnabled;
-
+    const activeAddressRoles = app.roles.getAllRolesInCommunity({
+      chain: app.activeChainId(),
+    });
+    const currentChainInfo = app.chain?.meta;
+    const hideChat =
+      currentChainInfo &&
+      activeAddressRoles &&
+      isActiveAddressPermitted(
```

Comment:
This doesn't feel clean to me -- makes more sense on server. On client, can we have it return a boolean instead?

Reply:
yes!